host/xtest/asym_perf.c

```diff
@@ -904,6 +904,25 @@ int asym_perf_runner_cmd_parser(int argc, char *argv[])
 				main_algo = ALGO_X25519;
 				width_bits = 256;
 				mode = MODE_GENKEYPAIR;
+			} else if (!strcasecmp(argv[i], "SM2_GENKEYPAIR")) {
```

Comment:
Don't you need to provide the `main_algo` info to generate an SM2 keypair?

Reply:
sloved

Reply:
solved